Exiled.CustomRoles/API/Features/CustomAbility.cs

```diff
@@ -60,18 +60,38 @@ public CustomAbility()
         public string AbilityType { get; }
 
         /// <summary>
-        /// Gets a <see cref="CustomRole"/> by name.
+        /// Gets a <see cref="CustomAbility"/> by name.
         /// </summary>
-        /// <param name="name">The name of the role to get.</param>
-        /// <returns>The role, or <see langword="null"/> if it doesn't exist.</returns>
+        /// <param name="name">The name of the ability to get.</param>
+        /// <returns>The ability, or <see langword="null"/> if it doesn't exist.</returns>
         public static CustomAbility? Get(string name) => Registered?.FirstOrDefault(r => r.Name == name);
 
         /// <summary>
-        /// Tries to get a <see cref="CustomRole"/> by name.
+        /// Gets a <see cref="CustomAbility"/> by type.
         /// </summary>
-        /// <param name="name">The name of the role to get.</param>
-        /// <param name="customAbility">The custom role.</param>
-        /// <returns>True if the role exists.</returns>
+        /// <param name="type">The type of the ability to get.</param>
+        /// <returns>The type, or <see langword="null"/> if it doesn't exist.</returns>
+        public static CustomAbility? Get(Type type) => Registered?.FirstOrDefault(r => r.GetType() == type);
+
+        /// <summary>
+        /// Tries to get a <see cref="CustomAbility"/> by type.
+        /// </summary>
+        /// <param name="type">The type of the ability to get.</param>
+        /// <param name="customAbility">The custom ability.</param>
+        /// <returns>True if the ability exists, otherwise false.</returns>
+        public static bool TryGet(Type type, out CustomAbility? customAbility)
+        {
+            customAbility = Get(type);
+
+            return customAbility is not null;
```

Comment:
can't this be inlined ?

Reply:
It *can* but there's no benefit of it, and doesn't follow the style consistency of the various existing TryGet methods in the API.